### ISSUE TITLE
Including particle-less forms of expressions as sK forms e.g. 贅沢言う in the 贅沢を言う entry

### ISSUE THREAD

**Issue by Marcusjmdict** [JMdictProject/JMdictIssues#134]
I think we ought to add forms like 贅沢言う to entries like 贅沢を言う.

贅沢を言う      	9357	82.6%
贅沢をいう      	1596	14.1%
ぜいたくを言う	369  	3.3%

贅沢言う         	11762	80.8%
贅沢いう         	2003	13.8%
ぜいたく言う  	790  	5.4%

**Comment by JMdictProject** [comment]
I see GG5 even uses it in an example:
もらっておきながら贅沢言うな． It's a bit much to say that after you've accepted it.

I suspect it has a degree of informality cf the full term, but I don't have a problem with adding them as [sK] forms. Probably better than not having them, or having parallel entries.

**Comment by Marcusjmdict** [comment]
We have more than 3000 entries with an を in them. Most of them would probably qualify.

**Comment by JMdictProject** [comment]
Can we agree:
- that it's OK to add them where they are common enough
- that we'll tag them as [sK]
If there's agreement on that, I'll close this issue.

**Comment by briankrznarich** [comment]
I agree with everything above.  I'll add that while を is simple, が・の particle-dropping(which seems just as valid as を-dropping) needs some additional consideration on where to put the particle-less form(が, の, both, most-popular entry...). 

With that in mind, I'd also like to ask: could we use [sK] to completely avoid duplicating が・の pairs? (perhaps not universally, but where it seems harmless)

For example, I quite coincidentally just ran into such a case:
人聞きの悪い	10207	63.8% <-- we have this (disreputable; scandalous; disgraceful)
人聞きが悪い	2850	17.8%  <-- not this
人聞き悪い	2932	18.3% <-- not this (this is what I saw in-the-wild)

Following Marcus' suggestion, we could stick 人聞き悪い[sK] into 人聞きの悪い.  With that out of the way, do we really need a separate entry for the が form: 人聞きが悪い ?  Or could we just [sK] them both into the existing entry and call it a day?

In the grand scheme, I think that in most cases we don't really benefit much from double-entering が・の pairs(we must have hundreds of them).  Sometimes it lets use give better English [adj-f] glosses, but often we end up with entries like:
人気がある [verb] to be popular
人気のある [adj-f,exp] popular

It's even less productive when the tail is an adjective to begin with.　For example, we have two *long* identical entries for:

つかみどころがない[exp,adj-f] | 8571 | 36.7%
つかみどころのない[exp,adj-f] | 14793 | 63.3%

If we kept the が form, and [sk]'d the の form into it, the existence of the の form should be grammatically self-evident. All we lose is the ability to specify whether の or が is more common, which I think is quite minor here. 

Getting back, for the particle-less  つかみどころない(420 ngrams), we could [sk] it into either or both forms above.  Technically I'd say "both" is the right answer, but it would be nice if there was just one entry for all of it.

This is all somewhat related to the discussion here, which is about condensing forms that differ only by grammatical transformation, while increasing discoverability in search results:
https://github.com/JMdictProject/JMdictIssues/issues/87

**Comment by CarlBN** [comment]
In [2078980](https://www.edrdg.org/jmwsgi/srchres.py?svc=jmdict&q=2078980&src=1&srchres=1) an anonymous user commented:

> here the を less version was included as an sK, but in 頭がおかしい the が less version was NOT included. Is there a policy on this?

I think we should quickly discuss how we want the three different cases to be handled.
Perhaps not as an absolute guideline, but instead to simplify some case-by-case decisions.

### Case 1 - full form is far more common
E.g. the 虚勢を張る in question:

| Term | Ngrams |
|--------|--------|
| 虚勢を張る | 7,433 |
| 虚勢張る |  745 | 

As already clear, in these cases the particle-less form should just be added as a search key.

### Case 2 - both forms are +- equally common; or both are common, but the particle-less form is somewhat more common
E.g. the 頭がおかしい in question:

| Term | Ngrams |
|--------|--------|
| 頭がおかしい | 61010 |
| 頭おかしい | 92590 | 

This maybe isn't the best example, as I'd also probaby have given 頭おかしい its own entry.
And don't really want to draw a line in the sand and come up with a threshold, but maybe that's the easiest solution?
E.g. "as long as the particle-less form is less than 20% more common than the full form, it can still be included as an [sK]"?

### Case 3 - particle-less form is far more common
E.g. the recent 丹精を込める:

| Term | Ngrams |
|--------|--------|
| 丹精を込めて | 7,304 |
| 丹精込めて | 112,532 | 

I think in cases like that it's much more reasonable to do it the other way around. I.e. having the particle-less form be headword and the full form as a search key.

**Comment by JMdictProject** [comment]
In cases 1 and 3 I think it's reasonably clear; if the meanings are the same and we are seeing ratios of around 10:1 we'd agree on making the less common form a hidden search-only key of the more common form.

For case 2, where the forms are closer to each other in numbers, I think we'd be more likely to have separate entries. That's what was done with 頭がおかしい/頭おかしい. It would be a case by case decision really.
